src/store/modules/cells/actions.js

```diff
@@ -66,32 +66,32 @@ const actions = {
    * @param nDestRow - 1-based row of empty destination cell
    * @param nDestCol - 1-based column of empty destination cell
    */
-  aKingMovement({ commit, state }, payload) {
+  aKingMovement({ commit, rootState, state }, payload) {
     const { coords, isPlayerBlack } = payload
     commit('mKingMovement', coords)
-    writeBoardToDB(state.cells, isPlayerBlack)
+    writeBoardToDB(rootState.game.currentGameID, state.cells, isPlayerBlack)
   },
 
   /**
    * aCapturePiece moves a black or white chip to capture an opposite-colored piece
    * diagonally adjacent from it.
    * @param coords - an object containing the source and destination coordinates
    */
-  aCapturePiece({ commit, state }, payload) {
+  aCapturePiece({ commit, rootState, state }, payload) {
     const { coords, isPlayerBlack } = payload
     commit('mCapturePiece', coords)
-    writeBoardToDB(state.cells, isPlayerBlack)
+    writeBoardToDB(rootState.game.currentGameID, state.cells, isPlayerBlack)
   },
 
   /**
    * aKingCapturePiece moves a black or white king to capture an opposite-colored piece
    * in its diagonal
    * @param coords - an object containing the source and destination coordinates
    */
-  aKingCapturePiece({ commit, state }, payload) {
+  aKingCapturePiece({ commit, rootState, state }, payload) {
     const { coords, isPlayerBlack } = payload
     commit('mKingCapturePiece', coords)
-    writeBoardToDB(state.cells, isPlayerBlack)
+    writeBoardToDB(rootState.game.currentGameID, state.cells, isPlayerBlack)
```

Comment:
action.js changes are completely redundant with changes for pregame, will anticipate the merge conflicts XD